app/Http/Controllers/Api/AccessoriesController.php

```diff
@@ -316,19 +314,19 @@ public function checkout(AccessoryCheckoutRequest $request, Accessory $accessory
      */
     public function checkin(Request $request, $accessoryUserId = null)
     {
-        if (is_null($accessory_user = DB::table('accessories_users')->find($accessoryUserId))) {
+        if (is_null($accessory_checkout = DB::table('accessories_checkout')->find($accessoryUserId))) {
```

Comment:
Since we're already modifying this line, is there maybe a way for us to *not* say `DB::table()` here? That always feels to me like a code smell. And the line's already being altered, is there a way to do this properly polymorphic and make it a little cleaner?

Reply:
Adjusted